package.json

```diff
@@ -37,7 +37,7 @@
         "test-server": "jest --config jest.config.ssr.json",
         "test-projection": "yarn run collect-projection-tests && start-server-and-test 'python -m SimpleHTTPServer' http://localhost:8000 'yarn run cypress run -s cypress/integration/projection.ts --config baseUrl=http://0.0.0.0:8000/'",
         "test-e2e": "start-server-and-test start-dev-server http://localhost:9990 'cypress run --config ignoreTestFiles=projection.ts'",
-        "test-e2e-file": "start-server-and-test start-dev-server http://localhost:9990 'cypress run --spec \"cypress/integration/layout-shared.ts\"'",
+        "test-e2e-file": "start-server-and-test start-dev-server http://localhost:9990 'cypress run --spec \"cypress/integration/layout-resize.ts\"'",
```

Comment:
I think I mentioned this before, but should we remove this filename from the package.json?

Reply:
I don't think so, it makes it a lot easier to run just one test in this otherwise quite lengthy suite, and as it's in the interest of time prevents us having to rewrite the command each time

Reply:
Yeah, but what I ment is that it should be possible to pas this as an argument to the command, right?

Reply:
Ah I see, I'll have to look into it, I think it's already a little difficult to do this through package.json scripts but additionally this is like a script within a script (see the embedded quotation marks) so I'm not sure how you'd pass it through to that